src/components/SummaryConfirmation/index.js

```diff
@@ -24,32 +34,30 @@ const SummaryConfirmation = ({submissionAllowed, onPrevPage}) => {
 
   const {
     loading,
-    value: privacyPolicyConfig = {},
+    value: statementsInfo = [],
     error,
   } = useAsync(async () => {
-    if (!canSubmit) return undefined;
-    return await getPrivacyPolicyInfo(baseUrl);
-  }, [baseUrl, getPrivacyPolicyInfo, canSubmit]);
+    if (!canSubmit) return [];
+
+    return await getStatementsInfo(baseUrl);
+  }, [baseUrl, getStatementsInfo, canSubmit]);
+  const {values: formikValues} = useFormikContext();
+
   if (error) throw error;
 
-  const {requiresPrivacyConsent, privacyLabel} = privacyPolicyConfig;
-  const {
-    values: {privacy: policyAccepted},
-  } = useFormikContext();
+  const submitDisabled = !isSubmitEnabled(loading, statementsInfo, formikValues);
 
-  const displayPrivacyNotice = !loading && canSubmit && requiresPrivacyConsent;
-  const submitDisabled = requiresPrivacyConsent && !policyAccepted;
-  const [warningUncheckedPrivacy, setWarningUncheckedPrivacy] = useState(false);
-  if (policyAccepted && warningUncheckedPrivacy) {
-    setWarningUncheckedPrivacy(false);
-  }
+  const [showStatementWarnings, setShowStatementWarnings] = useState(false);
 
   return (
     <>
-      {displayPrivacyNotice && (
-        <PrivacyCheckbox label={privacyLabel} showWarning={warningUncheckedPrivacy} />
-      )}
       {loading && <Loader />}
+      <StatementCheckboxes
+        loading={loading}
+        canSubmit={canSubmit}
+        statementsInfo={statementsInfo}
+        showWarnings={showStatementWarnings}
+      />
```

Comment:
I think the checkboxes themselves don't really need to know about `canSubmit` and we could rather do something like:

```jsx
{canSubmit && <StatementCheckboxes ... />}
```

Probably we should also incorporate `loading` in here?